### ISSUE TITLE
Addig this ti gh0stzk's rice

### ISSUE THREAD

**Issue by Momen-MKadry** [adilhyz/dotfiles-v1#4]
Hey, so I found your rice and I liked it and wanted to add it to my already riced system, I followed the instructions in gh0stzk's wiki but the cursor and gtk theme and fonts are messed up now, is there a way to add your rices?

I added them by copying the rices from the `bspwm/rices` directory

**Comment by adilhyz** [comment]
yes basically, my rice is based on ghostzk here's how to install it

> [📦 Setup](https://github.com/adilhyz/dotfiles-v1?tab=readme-ov-file#-setup)
<details>
<summary><b>Installation:</b></summary>

The installer only works for **ARCH** Linux, and based distros.

<b>Open a terminal in HOME</b>
First download the [installer](https://github.com/adilhyz/dotfiles-v1/blob/master/RiceInstaller)
```sh
curl https://raw.githubusercontent.com/adilhyz/dotfiles-v1/master/RiceInstaller -o $HOME/RiceInstaller
```
Now give it execute permissions
```sh
chmod +x RiceInstaller
```
Finally run the [installer](RiceInstaller)
```sh
./RiceInstaller
```
</details>

I have also prepared several themes and icons which automatically run when installing rice.

[*repo="https://adilhyz.github.io/adilhyz-repo/x86_64/"
iconsu=("adilhyz-icons")
themesu=("adilhyz-themes")
cursorsu=("adilhyz-cursors")
piles=("-1.1-1-any.pkg.tar.zst"*)](https://github.com/adilhyz/dotfiles-v1/blob/209da9390a514d3e44e3f37b379af8448ccd32db/RiceInstaller#L111C1-L115C33)

font you can [visit](https://adilhyz.github.io/en/posts/arch-linux-full-install-with-bspwm/#35-installation-package-requirement) my article for my paclists ```adilhyz/archlinux```

simply for fonts [```adilhyz/archlinux/paclists/bspwm/fonts```](https://github.com/adilhyz/ArchLinux/blob/5af87f1572a722ec2f88e4e8975dd3ed0986f0be/Paclists/Bspwm/Fonts#L1C1-L17C23)


```
git clone https://github.com/adilhyz/ArchLinux AIO
```
```
pacman -S --needed - < /AIO/Paclists/Bspwm/fonts
```

**Comment by Momen-MKadry** [comment]
I want to add two themes to my current config, I have some edits on the gh0stzk rice, when I did copy the 2 'rices' folders, they showed up in the theme selector but the mouse and font were broken across all themes

**Comment by adilhyz** [comment]
### Opinion
I would argue that gh0stzk basically doesn't support automatic mouse and theme switching yet, that's why I added mouse and theme icon switching as well. as I explained before

> switching script:

<details>
<summary> vars </summary>

bspwm/Rices/all/`Theme.sh`

```sh
set_appearance() {
	XFILE="$BDIR/xsettingsd"
	GTK2FILE="$HOME/.gtkrc-2.0"
	GTK3FILE="$HOME/.config/gtk-3.0/settings.ini"

	# apply gtk theme, icons, cursor & fonts
	if [[ `pidof xsettingsd` ]]; then
		sed -i -e "s|Net/ThemeName .*|Net/ThemeName \"amarena\"|g" ${XFILE}
		sed -i -e "s|Net/IconThemeName .*|Net/IconThemeName \"Nordzy-pink-dark\"|g" ${XFILE}
		sed -i -e "s|Gtk/CursorThemeName .*|Gtk/CursorThemeName \"phinger-cursors\"|g" ${XFILE}
	else
		sed -i -e "s/gtk-font-name=.*/gtk-font-name=\"Noto Sans 9\"/g" ${GTK2FILE}
		sed -i -e "s/gtk-theme-name=.*/gtk-theme-name=\"amarena\"/g" ${GTK2FILE}
		sed -i -e "s/gtk-icon-theme-name=.*/gtk-icon-theme-name=\"Nordzy-pink-dark\"/g" ${GTK2FILE}
		sed -i -e "s/gtk-cursor-theme-name=.*/gtk-cursor-theme-name=\"phinger-cursors\"/g" ${GTK2FILE}
		
		sed -i -e "s/gtk-font-name=.*/gtk-font-name=Noto Sans 9/g" ${GTK3FILE}
		sed -i -e "s/gtk-theme-name=.*/gtk-theme-name=amarena/g" ${GTK3FILE}
		sed -i -e "s/gtk-icon-theme-name=.*/gtk-icon-theme-name=Nordzy-pink-dark/g" ${GTK3FILE}
		sed -i -e "s/gtk-cursor-theme-name=.*/gtk-cursor-theme-name=phinger-cursors/g" ${GTK3FILE}
	fi
	
	# inherit cursor theme
	if [[ -f "$HOME"/.icons/default/index.theme ]]; then
		sed -i -e "s/Inherits=.*/Inherits=phinger-cursors/g" "$HOME"/.icons/default/index.theme
	fi	
}

set_appearance
```

bspwm/`bspwmrc`

```
# Lauch xsettingsd daemon
xsettingsd --config="$HOME"/.config/bspwm/xsettingsd &
```

> `xsettingsd` pkgs required
>
> https://github.com/adilhyz/dotfiles-v1/blob/209da9390a514d3e44e3f37b379af8448ccd32db/RiceInstaller#L84

</details>

### speculation

But I guess this doesn't answer the question [_the mouse and font were broken across all themes_](https://github.com/adilhyz/dotfiles-v1/issues/4#issuecomment-2057025753), maybe your rice is broken in some edits? have an example let me help you
